.github/workflows/swift_package_test.yml

```diff
@@ -84,7 +87,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        swift_version: ['5.9', '5.10', '6.0', 'nightly', 'nightly-6.0']
+        swift_version: ['5.9', '6.0', 'nightly', 'nightly-6.0']
```

Comment:
We should add 5.10 back

Reply:
In our matrix we just said Windows is 6.0+. Since we never had CI for Windows on our packages that makes our lives easier and we don't have to chase old toolchain bugs.